server/ntb/publish/ntb_ninjs.py

```diff
@@ -23,46 +17,94 @@ def __init__(self):
         super().__init__()
         self.format_type = "ntb_ninjs"
 
-    def format(self, article, subscriber, codes=None):
-        return super().format(article, subscriber, codes)
-
     def _transform_to_ninjs(self, article, subscriber, recursive=True):
         ninjs = super()._transform_to_ninjs(article, subscriber, recursive)
-        ninjs["version"] = str(article.get("correction_sequence", 1))
 
-        # mapping Imatrics
-        if article.get("person"):
-            ninjs["people"] = self.format_people(article)
+        ninjs["people"] = self.format_people(article)
 
-        if article.get("organisation"):
-            ninjs["organisations"] = self.format_organisation(article)
+        ninjs["organisations"] = self.format_organisations(article)
+
+        if ninjs.get("headline"):
+            ninjs["headlines"] = self.format_headlines(article)
+
+        if ninjs.get("description_text"):
+            ninjs["descriptions"] = self.format_descriptions(ninjs)
+
+        if ninjs.get("body_html"):
+            ninjs["bodies"] = self.format_bodies(ninjs)
+
+        if ninjs.get("subject"):
+            ninjs["subjects"] = self.format_subjects(ninjs)
+
+        # removable items which mapped according to Ninjs v2
+        removable_items = [
```

Comment:
might be better to work with whitelist here so when there are some changes in the core ninjs output we don't have to modify this

Reply:
What whitelist could you please explain me about this whitelist ?

Reply:
I mean the properties from https://iptc.org/std/ninjs/ninjs-schema_2.0.json

Reply:
so you mean we have to store the properties according to schema in a new variable from core ninjs output ?

Reply:
I mean that instead of using blacklist `removable_items` we could have there `ninjs2_properties` and remove everything that's not in there:
```py
ninjs_properties = ['headlines', 'uri', ...]
for key in ninjs:
    if key not in ninjs_properties:
        ninjs.pop(key)
```